src/components/Button/Button.stories.tsx

```diff
@@ -57,18 +57,18 @@ PrimaryInverted.args = {
   text: 'Primary Button',
 };
 
-export const BareIcon = Template.bind({});
-BareIcon.args = {
-  variant: 'bare',
+export const IconButton = Template.bind({});
```

Comment:
I only updated the story names and types; will add stories for the new variants in a follow-up PR.